### PR TITLE
[CH225] Hotfix dapp engineer Fix REFERRAL AGREEMENT

### PR DESCRIPTION
**Pull request by 92Sam**
[CH225] https://app.clubhouse.io/masterventures/story/225/as-a-dapp-engineer-i-need-to-fix-the-issues-on-the-referral-agreement-template

FIX template Referal Agreement

### CODE REVIEW

File: redux/actions/template/index.ts [hunk 1]
```diff
@@ -797,14 +786,23 @@ const getContractTemplate = (
               type: 'string',
               format: 'date',
             },
+            titleProvider: {
+              title: 'Tittle',
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:790]: Not big issue; `Tittle` <= `Title`

File: redux/actions/template/index.ts [hunk 2]
```diff
@@ -797,14 +786,23 @@ const getContractTemplate = (
...
           // required: ['commisionDate'],
           required: ['date'],
         },
+
         {
           type: 'object',
           title: 'My information (the “Company)',
           properties: {
+            titleCompany: {
+              title: 'Tittle',
```
Comment (CoolHandLuke88) [inline redux/actions/template/index.ts:807]: same, Not big issue; Tittle <= Title

**Review by CoolHandLuke88 — APPROVED** [review]
+1 Address/Ignore comments

File: redux/actions/template/index.ts (hunk 1 above)
Comment (92Sam) [reply redux/actions/template/index.ts:790]: oh my bad fixing this into the new PR my friend

File: redux/actions/template/index.ts (hunk 2 above)
Comment (92Sam) [reply redux/actions/template/index.ts:807]: This another one into the next PR bro

**Pull request merged**